### ISSUE TITLE
Pre-release tasks (10.45-RC1)

### ISSUE THREAD

**Issue by NWilson** [PCRE2Project/pcre2#631]
- [x] Bump the libtool version number
- Check documentation files
    - [x] NEWS
    - [x] ChangeLog
    - [x] README
    - [x] NON-AUTOTOOLS-BUILD
- [x] Check/update last-modified times in manpages
- [x] Compare the 10.45 and 10.44 tarballs (`ls -l` and diff the output)

**Comment by NWilson** [comment]
Diff between the contents of the 10.44 and the 10.45-DEV tarballs:

(All these changes look acceptable/expected to me.)

```diff
--- pcre2-10.44.manifest	2024-12-18 14:45:52.113161737 +0000
+++ pcre2-10.45-DEV.manifest	2024-12-18 14:45:53.193161692 +0000
@@ -1,43 +1,79 @@
 drwxr-xr-x .
--rwxr-xr-x ./132html
--rw-r--r-- ./AUTHORS
+-rw-r--r-- ./AUTHORS.md
+-rw-r--r-- ./BUILD.bazel
 -rw-r--r-- ./CMakeLists.txt
 -rw-r--r-- ./COPYING
 -rw-r--r-- ./ChangeLog
--rwxr-xr-x ./CheckMan
--rwxr-xr-x ./CleanTxt
--rwxr-xr-x ./Detrail
 -rw-r--r-- ./HACKING
 -rw-r--r-- ./INSTALL
--rw-r--r-- ./LICENCE
+-rw-r--r-- ./LICENCE.md
+-rw-r--r-- ./MODULE.bazel
 -rw-r--r-- ./Makefile.am
 -rw-r--r-- ./Makefile.in
 -rw-r--r-- ./NEWS
 -rw-r--r-- ./NON-AUTOTOOLS-BUILD
--rwxr-xr-x ./PrepareRelease
 -rw-r--r-- ./README
 -rwxr-xr-x ./RunGrepTest
 -rw-r--r-- ./RunGrepTest.bat
 -rwxr-xr-x ./RunTest
 -rw-r--r-- ./RunTest.bat
+-rw-r--r-- ./SECURITY.md
+-rw-r--r-- ./WORKSPACE.bazel
 -rw-r--r-- ./aclocal.m4
 -rwxr-xr-x ./ar-lib
+-rw-r--r-- ./build.zig
 drwxr-xr-x ./cmake
 -rw-r--r-- ./cmake/COPYING-CMAKE-SCRIPTS
 -rw-r--r-- ./cmake/FindEditline.cmake
--rw-r--r-- ./cmake/FindPackageHandleStandardArgs.cmake
 -rw-r--r-- ./cmake/FindReadline.cmake
 -rw-r--r-- ./cmake/pcre2-config-version.cmake.in
 -rw-r--r-- ./cmake/pcre2-config.cmake.in
 -rwxr-xr-x ./compile
 -rw-r--r-- ./config-cmake.h.in
 -rwxr-xr-x ./config.guess
 -rwxr-xr-x ./config.sub
 -rwxr-xr-x ./configure
 -rw-r--r-- ./configure.ac
 -rwxr-xr-x ./depcomp
+drwxr-xr-x ./deps
+drwxr-xr-x ./deps/sljit
+drwxr-xr-x ./deps/sljit/sljit_src
+drwxr-xr-x ./deps/sljit/sljit_src/allocator_src
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitExecAllocatorApple.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitExecAllocatorCore.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitExecAllocatorFreeBSD.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitExecAllocatorPosix.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitExecAllocatorWindows.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitProtExecAllocatorNetBSD.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitProtExecAllocatorPosix.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitWXExecAllocatorPosix.c
+-rw-r--r-- ./deps/sljit/sljit_src/allocator_src/sljitWXExecAllocatorWindows.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitConfig.h
+-rw-r--r-- ./deps/sljit/sljit_src/sljitConfigCPU.h
+-rw-r--r-- ./deps/sljit/sljit_src/sljitConfigInternal.h
+-rw-r--r-- ./deps/sljit/sljit_src/sljitLir.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitLir.h
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeARM_32.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeARM_64.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeARM_T2_32.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeLOONGARCH_64.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeMIPS_32.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeMIPS_64.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeMIPS_common.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativePPC_32.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativePPC_64.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativePPC_common.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeRISCV_32.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeRISCV_64.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeRISCV_common.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeS390X.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeX86_32.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeX86_64.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitNativeX86_common.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitSerialize.c
+-rw-r--r-- ./deps/sljit/sljit_src/sljitUtils.c
 drwxr-xr-x ./doc
 drwxr-xr-x ./doc/html
 -rw-r--r-- ./doc/html/NON-AUTOTOOLS-BUILD.txt
 -rw-r--r-- ./doc/html/README.txt
 -rw-r--r-- ./doc/html/index.html
@@ -101,14 +137,16 @@
 -rw-r--r-- ./doc/html/pcre2_set_max_pattern_compiled_length.html
 -rw-r--r-- ./doc/html/pcre2_set_max_pattern_length.html
 -rw-r--r-- ./doc/html/pcre2_set_max_varlookbehind.html
 -rw-r--r-- ./doc/html/pcre2_set_newline.html
 -rw-r--r-- ./doc/html/pcre2_set_offset_limit.html
+-rw-r--r-- ./doc/html/pcre2_set_optimize.html
 -rw-r--r-- ./doc/html/pcre2_set_parens_nest_limit.html
 -rw-r--r-- ./doc/html/pcre2_set_recursion_limit.html
 -rw-r--r-- ./doc/html/pcre2_set_recursion_memory_management.html
 -rw-r--r-- ./doc/html/pcre2_set_substitute_callout.html
+-rw-r--r-- ./doc/html/pcre2_set_substitute_case_callout.html
 -rw-r--r-- ./doc/html/pcre2_substitute.html
 -rw-r--r-- ./doc/html/pcre2_substring_copy_byname.html
 -rw-r--r-- ./doc/html/pcre2_substring_copy_bynumber.html
 -rw-r--r-- ./doc/html/pcre2_substring_free.html
 -rw-r--r-- ./doc/html/pcre2_substring_get_byname.html
@@ -136,11 +174,10 @@
 -rw-r--r-- ./doc/html/pcre2sample.html
 -rw-r--r-- ./doc/html/pcre2serialize.html
 -rw-r--r-- ./doc/html/pcre2syntax.html
 -rw-r--r-- ./doc/html/pcre2test.html
 -rw-r--r-- ./doc/html/pcre2unicode.html
--rw-r--r-- ./doc/index.html.src
 -rw-r--r-- ./doc/pcre2-config.1
 -rw-r--r-- ./doc/pcre2-config.txt
 -rw-r--r-- ./doc/pcre2.3
 -rw-r--r-- ./doc/pcre2.txt
 -rw-r--r-- ./doc/pcre2_callout_enumerate.3
@@ -201,14 +238,16 @@
 -rw-r--r-- ./doc/pcre2_set_max_pattern_compiled_length.3
 -rw-r--r-- ./doc/pcre2_set_max_pattern_length.3
 -rw-r--r-- ./doc/pcre2_set_max_varlookbehind.3
 -rw-r--r-- ./doc/pcre2_set_newline.3
 -rw-r--r-- ./doc/pcre2_set_offset_limit.3
+-rw-r--r-- ./doc/pcre2_set_optimize.3
 -rw-r--r-- ./doc/pcre2_set_parens_nest_limit.3
 -rw-r--r-- ./doc/pcre2_set_recursion_limit.3
 -rw-r--r-- ./doc/pcre2_set_recursion_memory_management.3
 -rw-r--r-- ./doc/pcre2_set_substitute_callout.3
+-rw-r--r-- ./doc/pcre2_set_substitute_case_callout.3
 -rw-r--r-- ./doc/pcre2_substitute.3
 -rw-r--r-- ./doc/pcre2_substring_copy_byname.3
 -rw-r--r-- ./doc/pcre2_substring_copy_bynumber.3
 -rw-r--r-- ./doc/pcre2_substring_free.3
 -rw-r--r-- ./doc/pcre2_substring_get_byname.3
@@ -243,11 +282,11 @@
 -rwxr-xr-x ./install-sh
 -rw-r--r-- ./libpcre2-16.pc.in
 -rw-r--r-- ./libpcre2-32.pc.in
 -rw-r--r-- ./libpcre2-8.pc.in
 -rw-r--r-- ./libpcre2-posix.pc.in
--rw-r--r-- ./ltmain.sh
+-rwxr-xr-x ./ltmain.sh
 drwxr-xr-x ./m4
 -rw-r--r-- ./m4/ax_pthread.m4
 -rw-r--r-- ./m4/libtool.m4
 -rw-r--r-- ./m4/ltoptions.m4
 -rw-r--r-- ./m4/ltsugar.m4
@@ -264,10 +303,12 @@
 -rw-r--r-- ./src/pcre2.h.in
 -rw-r--r-- ./src/pcre2_auto_possess.c
 -rw-r--r-- ./src/pcre2_chartables.c.dist
 -rw-r--r-- ./src/pcre2_chkdint.c
 -rw-r--r-- ./src/pcre2_compile.c
+-rw-r--r-- ./src/pcre2_compile.h
+-rw-r--r-- ./src/pcre2_compile_class.c
 -rw-r--r-- ./src/pcre2_config.c
 -rw-r--r-- ./src/pcre2_context.c
 -rw-r--r-- ./src/pcre2_convert.c
 -rw-r--r-- ./src/pcre2_dfa_match.c
 -rw-r--r-- ./src/pcre2_dftables.c
@@ -275,10 +316,11 @@
 -rw-r--r-- ./src/pcre2_extuni.c
 -rw-r--r-- ./src/pcre2_find_bracket.c
 -rw-r--r-- ./src/pcre2_fuzzsupport.c
 -rw-r--r-- ./src/pcre2_internal.h
 -rw-r--r-- ./src/pcre2_intmodedep.h
+-rw-r--r-- ./src/pcre2_jit_char_inc.h
 -rw-r--r-- ./src/pcre2_jit_compile.c
 -rw-r--r-- ./src/pcre2_jit_match.c
 -rw-r--r-- ./src/pcre2_jit_misc.c
 -rw-r--r-- ./src/pcre2_jit_neon_inc.h
 -rw-r--r-- ./src/pcre2_jit_simd_inc.h
@@ -298,63 +340,32 @@
 -rw-r--r-- ./src/pcre2_substring.c
 -rw-r--r-- ./src/pcre2_tables.c
 -rw-r--r-- ./src/pcre2_ucd.c
 -rw-r--r-- ./src/pcre2_ucp.h
 -rw-r--r-- ./src/pcre2_ucptables.c
+-rw-r--r-- ./src/pcre2_util.h
 -rw-r--r-- ./src/pcre2_valid_utf.c
 -rw-r--r-- ./src/pcre2_xclass.c
 -rw-r--r-- ./src/pcre2demo.c
 -rw-r--r-- ./src/pcre2grep.c
 -rw-r--r-- ./src/pcre2posix.c
 -rw-r--r-- ./src/pcre2posix.h
 -rw-r--r-- ./src/pcre2posix_test.c
 -rw-r--r-- ./src/pcre2test.c
-drwxr-xr-x ./src/sljit
-drwxr-xr-x ./src/sljit/allocator_src
--rw-r--r-- ./src/sljit/allocator_src/sljitExecAllocatorApple.c
--rw-r--r-- ./src/sljit/allocator_src/sljitExecAllocatorCore.c
--rw-r--r-- ./src/sljit/allocator_src/sljitExecAllocatorFreeBSD.c
--rw-r--r-- ./src/sljit/allocator_src/sljitExecAllocatorPosix.c
--rw-r--r-- ./src/sljit/allocator_src/sljitExecAllocatorWindows.c
--rw-r--r-- ./src/sljit/allocator_src/sljitProtExecAllocatorNetBSD.c
--rw-r--r-- ./src/sljit/allocator_src/sljitProtExecAllocatorPosix.c
--rw-r--r-- ./src/sljit/allocator_src/sljitWXExecAllocatorPosix.c
--rw-r--r-- ./src/sljit/allocator_src/sljitWXExecAllocatorWindows.c
--rw-r--r-- ./src/sljit/sljitConfig.h
--rw-r--r-- ./src/sljit/sljitConfigCPU.h
--rw-r--r-- ./src/sljit/sljitConfigInternal.h
--rw-r--r-- ./src/sljit/sljitLir.c
--rw-r--r-- ./src/sljit/sljitLir.h
--rw-r--r-- ./src/sljit/sljitNativeARM_32.c
--rw-r--r-- ./src/sljit/sljitNativeARM_64.c
--rw-r--r-- ./src/sljit/sljitNativeARM_T2_32.c
--rw-r--r-- ./src/sljit/sljitNativeLOONGARCH_64.c
--rw-r--r-- ./src/sljit/sljitNativeMIPS_32.c
--rw-r--r-- ./src/sljit/sljitNativeMIPS_64.c
--rw-r--r-- ./src/sljit/sljitNativeMIPS_common.c
--rw-r--r-- ./src/sljit/sljitNativePPC_32.c
--rw-r--r-- ./src/sljit/sljitNativePPC_64.c
--rw-r--r-- ./src/sljit/sljitNativePPC_common.c
--rw-r--r-- ./src/sljit/sljitNativeRISCV_32.c
--rw-r--r-- ./src/sljit/sljitNativeRISCV_64.c
--rw-r--r-- ./src/sljit/sljitNativeRISCV_common.c
--rw-r--r-- ./src/sljit/sljitNativeS390X.c
--rw-r--r-- ./src/sljit/sljitNativeX86_32.c
--rw-r--r-- ./src/sljit/sljitNativeX86_64.c
--rw-r--r-- ./src/sljit/sljitNativeX86_common.c
--rw-r--r-- ./src/sljit/sljitSerialize.c
--rw-r--r-- ./src/sljit/sljitUtils.c
 -rwxr-xr-x ./test-driver
 drwxr-xr-x ./testdata
 -rw-r--r-- ./testdata/grepbinary
 -rw-r--r-- ./testdata/grepfilelist
 -rw-r--r-- ./testdata/grepinput
 -rw-r--r-- ./testdata/grepinput3
 -rw-r--r-- ./testdata/grepinput8
+-rw-r--r-- ./testdata/grepinputBad8
+-rw-r--r-- ./testdata/grepinputBad8_Trail
 -rw-r--r-- ./testdata/grepinputC.bz2
 -rw-r--r-- ./testdata/grepinputC.gz
 -rw-r--r-- ./testdata/grepinputM
+-rw-r--r-- ./testdata/grepinputUN
 -rw-r--r-- ./testdata/grepinputv
 -rw-r--r-- ./testdata/grepinputx
 -rw-r--r-- ./testdata/greplist
 -rw-r--r-- ./testdata/grepnot.bz2
 -rw-r--r-- ./testdata/grepoutput
@@ -386,10 +397,11 @@
 -rw-r--r-- ./testdata/testinput22
 -rw-r--r-- ./testdata/testinput23
 -rw-r--r-- ./testdata/testinput24
 -rw-r--r-- ./testdata/testinput25
 -rw-r--r-- ./testdata/testinput26
+-rw-r--r-- ./testdata/testinput27
 -rw-r--r-- ./testdata/testinput3
 -rw-r--r-- ./testdata/testinput4
 -rw-r--r-- ./testdata/testinput5
 -rw-r--r-- ./testdata/testinput6
 -rw-r--r-- ./testdata/testinput7
@@ -420,10 +432,11 @@
 -rw-r--r-- ./testdata/testoutput22-8
 -rw-r--r-- ./testdata/testoutput23
 -rw-r--r-- ./testdata/testoutput24
 -rw-r--r-- ./testdata/testoutput25
 -rw-r--r-- ./testdata/testoutput26
+-rw-r--r-- ./testdata/testoutput27
 -rw-r--r-- ./testdata/testoutput3
 -rw-r--r-- ./testdata/testoutput3A
 -rw-r--r-- ./testdata/testoutput3B
 -rw-r--r-- ./testdata/testoutput4
 -rw-r--r-- ./testdata/testoutput5
```

**Comment by NWilson** [comment]
And, the diff of the files installed by `make install`:

```diff
--- pcre2-10.44.manifest	2024-12-18 14:57:29.303134620 +0000
+++ pcre2-10.45-DEV.manifest	2024-12-18 14:57:29.903134467 +0000
@@ -23,16 +23,17 @@
 -rw-r--r-- install-dir/usr/local/lib/pkgconfig/libpcre2-8.pc
 -rw-r--r-- install-dir/usr/local/lib/pkgconfig/libpcre2-posix.pc
 drwxr-xr-x install-dir/usr/local/share
 drwxr-xr-x install-dir/usr/local/share/doc
 drwxr-xr-x install-dir/usr/local/share/doc/pcre2
--rw-r--r-- install-dir/usr/local/share/doc/pcre2/AUTHORS
+-rw-r--r-- install-dir/usr/local/share/doc/pcre2/AUTHORS.md
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/COPYING
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/ChangeLog
--rw-r--r-- install-dir/usr/local/share/doc/pcre2/LICENCE
+-rw-r--r-- install-dir/usr/local/share/doc/pcre2/LICENCE.md
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/NEWS
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/README
+-rw-r--r-- install-dir/usr/local/share/doc/pcre2/SECURITY.md
 drwxr-xr-x install-dir/usr/local/share/doc/pcre2/html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/NON-AUTOTOOLS-BUILD.txt
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/README.txt
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/index.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2-config.html
@@ -95,14 +96,16 @@
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_max_pattern_compiled_length.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_max_pattern_length.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_max_varlookbehind.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_newline.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_offset_limit.html
+-rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_optimize.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_parens_nest_limit.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_recursion_limit.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_recursion_memory_management.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_substitute_callout.html
+-rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_set_substitute_case_callout.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_substitute.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_substring_copy_byname.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_substring_copy_bynumber.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_substring_free.html
 -rw-r--r-- install-dir/usr/local/share/doc/pcre2/html/pcre2_substring_get_byname.html
@@ -201,14 +204,16 @@
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_max_pattern_compiled_length.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_max_pattern_length.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_max_varlookbehind.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_newline.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_offset_limit.3
+-rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_optimize.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_parens_nest_limit.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_recursion_limit.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_recursion_memory_management.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_substitute_callout.3
+-rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_set_substitute_case_callout.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_substitute.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_substring_copy_byname.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_substring_copy_bynumber.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_substring_free.3
 -rw-r--r-- install-dir/usr/local/share/man/man3/pcre2_substring_get_byname.3
```

**Comment by zherczeg** [comment]
Nice!

**Comment by NWilson** [comment]
Great. I'm very very close to being ready for RC1. I was even hoping, maybe, to release it for Christmas.

There are three tasks remaining that I'm tracking:
* Update NEWS and ChangeLog
* Carlo asked for a substantially different API to pcre2_set_substitute_case_callout #564
* Alex has an outstanding very minor regression: #526

From my point of view, there's nothing else that would be absolutely required for an RC1.

**Comment by NWilson** [comment]
We can make a branch for this - so that regular work for the next release can continue in between 10.45-RC1 and 10.45 (final). There's no big disruption to just freezing things on a branch and calling it "done".

**Comment by carenas** [comment]
> There's no big disruption to just freezing things on a branch

It might be, if the branch is called (or assumed to be): `maint-10.45`

eventually we will have to backport some fixes and release the first ever "stable maintenance" version of PCRE2 as 10.45.1

**Comment by NWilson** [comment]
I would expect to only release 10.45, 10.46, and so on. It would certainly be imaginable that after releasing 10.45, we make a load of changes in `master` and then after a few weeks a really critical bug report arrives. In that case, we would release 10.46 with the bugfix, but branched off the 10.45 tag.

On the other hand, if we have already released 10.46, and a critical bug report comes in for 10.45, then we fix-forward and release 10.47 based off 10.46 (rather than trying to insert a release in between 10.45 and 10.46).

For the size of our team, that's the only approach we can manage I think. We don't want two parallel branches, both receiving releases concurrently.

Anyway, it's all hypothetical. We'll just fix things case-by-case as they arrive.
